pcdsdevices/valve.py

```diff
@@ -250,7 +250,7 @@ class VFS(Device):
                          doc='Fast Shutter Closed Valve Position')
     position_open = Cpt(EpicsSignalRO, ':IS_OPEN_RBV', kind='normal',
                         doc='Fast Shutter Open Valve Position')
-    vac_fault_ok = Cpt(EpicsSignalRO, ':FAULT_OK_RBV', kind='normal',
+    vac_fault_ok = Cpt(EpicsSignalRO, ':VAC_FAULT_OK_RBV', kind='normal',
                        doc=('Fast Shutter Vacuum Fault OK. To open VFS, this'
                             'needs to be reset to TRUE after a vacuum event'))
```

Comment:
I'm a little confused by the doc for this component as it says it needs to be reset making it seems someone needs to write TRUE to it but it's a readonly, so it can't be written to. Can you clarify?

Reply:
Ahh, 

> We merged this last time under the assumption that all the PVs have been checked. Can you do a quick check before we merge this one?
> 
> ```
> source pcds_conda
> cd <your pcdsdevices clone>
> ipython
> from pcdsdevices.valve import VFS
> vfs = VFS('<your prefix>', name='vfs')
> vfs.get()
> ```
> 
> This should work successfully if all the PV names are valid.

Thank for this PV check, I'll add it to my toolbox! Yes the PV check works properly for the VFS class.

Reply:
I have updated the doc strings accordingly. Thank you all for your discerning eyes!